classpath.gradle

```diff
@@ -8,7 +8,7 @@ task classpath << {
     def listFiles = new File(exploded)
     if (listFiles.exists()) {
       listFiles.eachFileRecurse(){ file ->
-        if (file.name.endsWith("classes.jar")){
+        if (file.name.endsWith(".jar")){
```

Comment:
In most Gradle projects (including Android) the build directory is used as a sort of temporary storage area for files that Gradle has processed. To get java to compile the resulting files into .class files, they are bundled into .jar files and were typically called "classes.jar." It is not a requirement though to name them "classes.jar" and that is why this change is needed - we were missing internal_impl-23.1.1.jar and in the future there may be more jars outside of the "classes.jar" convention

At least that's how I understand it